augur/application/schema/augur_full.sql

```diff
@@ -2777,6 +2777,35 @@ CREATE TABLE augur_data.working_commits (
 
 ALTER TABLE augur_data.working_commits OWNER TO augur;
 
+--
+-- Name: repo_clones_id_seq; Type: SEQUENCE; Schema: augur_data; Owner: augur
+--
+
+CREATE SEQUENCE augur_data.repo_clones_data_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
```

Comment:
Why are these changes put into `augur_full.sql` and not into a schema revision? I can't find anywhere `augur_full.sql` is even referenced

Reply:
I'm not sure. They came in from a fork. Perhaps it was old?